### PR TITLE
MTP-2548 apply external icon on Highlighted content and -links

### PR DESCRIPTION
**Pull request by petterkoding**

### CODE REVIEW

File: mt-kit/core/css/src/global/_links.scss [hunk 1]
```diff
@@ -427,6 +428,11 @@ a {
     @include apply-icon($icon-arrow-right-primary-small, before) {
       flex-shrink: 0;
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:429]: Did you check if you needed this? Looks to me like its not needed

File: mt-kit/core/css/src/global/_links.scss [hunk 2]
```diff
@@ -427,6 +428,11 @@ a {
...
     }
+    @include apply-external-link-icon($position: before) {
+      &::after {
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:432]: Did you check if you needed this? Looks to me like its not needed

**Review by lha047 — CHANGES_REQUESTED** [review]
See comments

File: mt-kit/core/css/src/global/_links.scss (hunk 1 above)
Comment (petterkoding) [reply mt-kit/core/css/src/global/_links.scss:429]: it is needed for mobile

image

File: mt-kit/core/css/src/global/_links.scss (hunk 2 above)
Comment (petterkoding) [reply mt-kit/core/css/src/global/_links.scss:432]: actually theres a  <p> wrapping the <a> in CardArticle.svelte. I can just remove the <p> Then there wont be a second icon.

**Review by lha047 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**